test_ir.py

```diff
@@ -18,5 +18,14 @@ class IRkernelTests(jkt.KernelTests):
     complete_code_samples = ['1', "print('hello, world')", "f <- function(x) {\n  x*2\n}"]
     incomplete_code_samples = ["print('hello", "f <- function(x) {\n  x*2"]
 
+class InstallspecTests(jkt.KernelTests):
+    # just a small test, it's the same kernel after all...
+    kernel_name = "testir"
+
+    language_name = "R"
+
+    code_hello_world = "print('hello, world')"
+
+
```

Comment:
yeah, that’s stupid. but i do accept tests failing because of it (at the end, so that you can see that everything else works)